c-cannons/bangazon-initial_site
# Hc sprint1 shopping cart

## Pull Request
#### Descriptions

1. Added functionality to display items in cart by logged-in customer for orders that are open.
1. Customer can also delete items in the cart.

#### Steps to Test

You must provide clear steps for any teammate to test the code.

1. `git fetch origin HC-sprint1-shopping-cart`
1. `git checkout HC-sprint1-shopping-cart`
1. Open db-browser and choose a test customer that has a sample order in the database
1.  In db-browser, navigate to the 'orders' table and set 'payment_method_id' for your selected order to NULL, and write changes
1. Log in to the app as your selected test customer
1. Navigate to Cart
1. Products should appear in the display with a link for "Delete from Cart"
1. When the customer clicks the "Delete from Cart" link, the product should disappear from the cart

#### Link to Feature Ticket

[#6](https://github.com/c-cannons/bangazon-initial_site/issues/6)
